project-fob/Controllers/AttendeeController.cs

```diff
@@ -27,24 +27,37 @@ public void Fob(string value)
             var gotvalue = HttpContext.Session.TryGetValue("meetingid", out var meetingIdValue);
 
             string byteArrayToString = System.Text.Encoding.ASCII.GetString(meetingIdValue);
-            Fob fob = db.Fob.Include(x => x.Meeting).ThenInclude(x => x.Stats).Include(x => x.fobbed).SingleOrDefault(f => f.Meeting.MeetingId == byteArrayToString);
+            Fob fob = db.Fob.Include(x => x.Meeting).ThenInclude(x => x.Stats).Include(x => x.fobbed).ThenInclude(x => x.User).SingleOrDefault(f => f.Meeting.MeetingId == byteArrayToString);
             if (fob == null)
             {
                 throw new ArgumentNullException();
```

Comment:
`ArgumentNullException` should really be thrown when one of the arguments to a method is set to `null` (and when that isn't allowed). Not finding something in a collection (which is what's happening here) often results in `throw new InvalidOperationException()`

However, just above this line `SingleOrDefault` is being used. That method means "give me the one element that matches some criteria, and if there is not exactly one element then return null". If instead you used `Single`, you won't need the null check. `Single` means "give me the one element that matches some criteria, and if there is not exactly one element then `throw InvalidOperationException`"